docs/ja/docs/advanced/nosql-databases.md

```diff
@@ -0,0 +1,155 @@
+# NoSQL (分散 / ビッグデータ) Databases
+
+**FastAPI** はあらゆる <abbr title="分散データベース (Big Data), や 'Not Only SQL'">NoSQL</abbr>と統合することもできます。
+
+ここでは<abbr title="ここでのドキュメントとは、キーと値を持つJSONオブジェクト（ディクショナリー）をあらわし、これらの値は他のJSONオブジェクトや配列（リスト）、数値、文字列、真偽値などにすることができます。">ドキュメント</abbr>ベースのNoSQLデータベースである**<a href="https://www.couchbase.com/" class="external-link" target="_blank">Couchbase</a>**を使用した例を見てみましょう。
+
+他にもこれらのNoSQLデータベースを利用することが出来ます:
+
+* **MongoDB**
+* **Cassandra**
+* **CouchDB**
+* **ArangoDB**
+* **ElasticSearch** など。
+
+!!! 豆知識
```

Comment:
```suggestion
!!! tip "豆知識"
```
I'm not sure but maybe this is (or was) right syntax.

Reply:
Thank you to suggests. I applied at here.
https://github.com/tiangolo/fastapi/pull/4205/commits/6866770bda9532a4860e7c6e661ab7efcba6a60f#diff-101eb49c79208a3498543c29b9170aaabf8ce9c8dbdc16b3eb63f4010c9081eeL15